modules/menu/menu.c

```diff
@@ -626,12 +626,30 @@ struct call *menu_find_call(const char *id)
  */
 void menu_selcall(struct call *call)
 {
+	int i;
+	enum call_state state[] = {
+		CALL_STATE_INCOMING,
+		CALL_STATE_OUTGOING,
+		CALL_STATE_RINGING,
+		CALL_STATE_EARLY,
+		CALL_STATE_ESTABLISHED,
+	};
+
 	menu.callid = mem_deref(menu.callid);
 
 	if (call) {
 		str_dup(&menu.callid, call_id(call));
 		call_set_current(ua_calls(call_get_ua(call)), call);
 	}
+	else {
+		for (i = sizeof(state)-1; i > 0; --i) {
```

Comment:
size(state) assumes that each element size is 1.

it is probably better to use `ARRAY_SIZE`